pkg/controller/pods.go

```diff
@@ -43,3 +48,26 @@ func (c *Controller) handlePodChange(obj interface{}) {
 	key := fmt.Sprintf("%s/%s", object.GetNamespace(), instanceName)
 	c.healthCheckQueue.AddAfter(key, 1*time.Second)
 }
+
+// DeletePodsByStatefulSet deletes all pods associated with a statefulset
+func (c *Controller) DeletePodsByStatefulSet(ctx context.Context, sts *appsv1.StatefulSet) (err error) {
+	listOpt := &client.ListOptions{
+		Namespace: sts.Namespace,
+	}
+	client.MatchingLabels(sts.Spec.Template.Labels).ApplyToList(listOpt)
+	podList := &corev1.PodList{}
+	err = c.k8sClient.List(ctx, podList, listOpt)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	if err := c.k8sClient.List(ctx, podList, listOpt); err != nil {
		return err
	}
```
just saving one line ;)